roles/openldap/tasks/password_policy.yml

```diff
@@ -0,0 +1,65 @@
+- name: check if ppolicy is part of the schema
+  shell: ldapsearch -Y EXTERNAL -H ldapi:/// -s sub
+    -b cn=config objectclass=olcSchemaConfig | grep ppolicy
+  register: ppolicy_out
+  ignore_errors: yes
+  changed_when: False
+
+- name: add password policy schema
+  shell: ldapadd -Y EXTERNAL -H ldapi:/// -f  /etc/ldap/schema/ppolicy.ldif
+  when: ppolicy_out.stdout.count("ppolicy,") < 1
+
+- name: create policy container
+  shell: ldapadd -x -D {{openldap.root_dn }} -w {{ openldap.root_password }} 
+    -f {{ openldap.config_dir }}/ppolicy_container.ldif
+  register: policy_command
+  failed_when: policy_command.rc != 0 and policy_command.rc != 68
+
+- name: check if ppolicy module is registered
+  shell: ldapsearch -Y EXTERNAL -H ldapi:/// -s sub 
+    -b olcDatabase={1}hdb,cn=config objectclass=olcPPolicyConfig
+  register: pm_out
+  ignore_errors: yes
+  changed_when: False
+
+- name: add password policy module
+  shell: ldapadd -Y EXTERNAL -H ldapi:/// -f {{ openldap.config_dir }}/ppolicy_module.ldif
+  when: pm_out.stdout.count(",ou={{ openldap.password_policy_ou }}") < 1
+
+- name: add default policy
+  shell: ldapadd -x -D {{openldap.root_dn }} -w {{ openldap.root_password }} 
+    -f {{ openldap.config_dir }}/password_policy.ldif
+  register: default_policy
+  failed_when: default_policy.rc != 0 and default_policy.rc != 68
+
+- name: add empty policy 
+  shell: ldapadd -x -D {{openldap.root_dn }} -w {{ openldap.root_password }}
+    -f {{ openldap.config_dir }}/empty_password_policy.ldif
+  register: empty_policy
+  failed_when: empty_policy.rc != 0 and empty_policy.rc != 68
+
+#TODO: Install from repo its uploaded there
+
+- name: copy the deb to a temp directory
+  copy: src=check_password/{{ ansible_distribution }}-{{ ansible_distribution_version }}-{{ ansible_userspace_bits }}-openldap-password-checker-1.0-1.deb
+        dest=/tmp/openldap-password-checker-1.0-1.deb
+
+- name: install checker from the package
+  shell: dpkg -i /tmp/openldap-password-checker-1.0-1.deb
+
+#END TODO Install from repo its uploaded there
+
+- name: add check_password ldif into ldap
+  command: /usr/bin/ldapmodify -x -D {{ openldap.root_dn }} 
+    -w {{ openldap.root_password }} -f {{ openldap.config_dir }}/add_passwd_config.ldif
+  register: check_pass
+  failed_when: check_pass.rc !=0 and check_pass.rc != 20
+
+- name: create check_password config directory
+  file: path=/etc/openldap 
+        state=directory
+
+- name: create check_password config
+  template: src=check_password.conf 
```

Comment:
template sources should match template destination. The source for this should be `etc/openldap/check_password.conf` and it should live in that relative path within the role's template directory.